src/main/java/dinkplugin/notifiers/data/SlayerNotificationData.java

```diff
@@ -2,11 +2,26 @@
 
 import lombok.EqualsAndHashCode;
 import lombok.Value;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 @Value
 @EqualsAndHashCode(callSuper = false)
 public class SlayerNotificationData extends NotificationData {
+
+    @NotNull
     String slayerTask;
+
+    @NotNull
     String slayerCompleted;
+
+    @NotNull
     String slayerPoints;
+
+    @Nullable // if jagex changes format of slayerTask
+    Integer incrementalKills;
```

Comment:
double-check if this name is good

Reply:
Updated in https://github.com/pajlads/DinkPlugin/pull/248/commits/15afb17ebe3b270e4391236c5ee49f258d416074